app/components/provider_interface/user_org_permission_breakdown_component.rb

```diff
@@ -0,0 +1,46 @@
+module ProviderInterface
+  class UserOrgPermissionBreakdownComponent < ViewComponent::Base
+    attr_reader :provider, :permission
+
+    def initialize(provider:, permission:)
+      @provider = provider
+      @permission = permission
+    end
+
+    def permitted_by_providers
```

Comment:
Could this be renamed to `partners_for_which_permission_applies` or similar? And the below could be `partners_for_which_permission_does_not_apply`